app/controllers/pensions/UpdatePensionProviderController.scala

```diff
@@ -239,4 +221,68 @@ class UpdatePensionProviderController @Inject()(taxAccountService: TaxAccountSer
 
       Future.successful(Ok(views.html.pensions.update.confirmation()))
   }
+
+  def redirectUpdatePension(id: Int): Action[AnyContent] = (authenticate andThen validatePerson).async {
```

Comment:
This stuff looks similar to the end employment journey, probably a dedupe refactor session could go on here